### DISCUSSION TITLE
Add adminPath option to keystone-next along with next.config.js overrides

### DISCUSSION THREAD

**Discussion by shendrick** [keystonejs/keystone#5862] [Feature requests]
Ideally, I'd like to run an SPA on the pages/index.js and have all of keystone at /admin. Right now that isn't possible because I can't set adminPath like I could with keystone 5. In addition, in order to get the SPA to take over routes, I'd need access to add to the next.config.js rewrites. Right now this is just copied from the keystone library with no way to append to it.

**Comment by bladey** [comment]
Thanks for the suggestion @shendrick!
